python/tvm/relay/frontend/onnx.py

```diff
@@ -1080,6 +1080,18 @@ class Or(Elemwise):
     def _impl_v7(cls, inputs, attr, params):
         return _op.logical_or(inputs[0], inputs[1])
 
+class Expand(OnnxOpConverter):
+    """ Operator converter for Expand.
+    """
+
+    @classmethod
+    def _impl_v1(cls, inputs, attr, params):
```

Comment:
Do we need to change the version number because the operator has been available since version 8 of the default ONNX operator set?

Reply:
+1. This method can be `_impl_v8`.

Reply:
Thanks, I'll fix the version.